retrofit-converters/wire/src/main/resources/META-INF/proguard/retrofit2-wire-converter.pro

```diff
@@ -0,0 +1,4 @@
+# ADAPTER field is looked up reflectively.
+-keepclassmembers class * extends com.squareup.wire.Message {
```

Comment:
I assume this doesn't prevent stripping the whole class if it's unused?

Reply:
Correct